apiary.apib

```diff
@@ -2643,6 +2645,23 @@ Download a credit note in a specific format.
             + location: `https://cdn.teamleader.eu/file` (string) - A temporary url where the requested file can be downloaded
             + expires: `2018-02-05T16:44:33+00:00` (string) - Expiration time of the temporary download link
 
+
+### invoices.credit [POST /invoices.credit]
+
+Credit an invoice completely.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + invoice_id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
```

Comment:
```suggestion
        + id: `d885e5d5-bacb-4607-bde9-abc4a04a901b` (string, required)
```

Reply:
Living in the creditNotes can't it be confusing being just id?